src/Components/Login.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { Container, Row, Col, Button, InputGroup, Carousel, Figure, Form } from 'react-bootstrap';
+
+class Login extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      isRemember: false,
+      passwordShown: false,
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.togglePasswordVisiblity = this.togglePasswordVisiblity.bind(this);
+  }
+  
+  handleInputChange(event) {
+    const target = event.target;
+    const value = target.name === 'isRemember' ? target.checked : target.value;
+    const name = target.name;
+
+    this.setState({
+      [name]: value
+    });
+  }
+
+  handleSubmit(event) {
+    alert('An email was submitted: ' + this.state.email);
+    event.preventDefault();
+  }
+
+  togglePasswordVisiblity(event) {
+    this.setState({
+      passwordShown: this.state.passwordShown ? false : true
+    });
+  };
+
+  render() {
+    
+    return (
+      <React.Fragment>
+        <Container className="homePage">
+          <Row className="vh-100">
+            <Col md={5} className="pr-5 my-auto loginForm">
+              <Figure>
+                <Figure.Image
+                  alt="171x180"
+                  src= "/rms_logo.jpg"
+                />
+              </Figure>
+              <Form className="my-3" onSubmit={this.handleSubmit}>
+                <Form.Group controlId="exampleForm.ControlInput1">
+                  <Form.Label>EMAIL</Form.Label>
+                  <InputGroup>
+                    <InputGroup.Prepend>
+                      <InputGroup.Text className="inputIcon">
+                        <svg className="bi bi-envelope" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                          <path fillRule="evenodd" d="M14 3H2a1 1 0 0 0-1 1v8a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1V4a1 1 0 0 0-1-1zM2 2a2 2 0 0 0-2 2v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V4a2 2 0 0 0-2-2H2z"/>
+                          <path d="M.05 3.555C.017 3.698 0 3.847 0 4v.697l5.803 3.546L0 11.801V12c0 .306.069.596.192.856l6.57-4.027L8 9.586l1.239-.757 6.57 4.027c.122-.26.191-.55.191-.856v-.2l-5.803-3.557L16 4.697V4c0-.153-.017-.302-.05-.445L8 8.414.05 3.555z"/>
+                        </svg>                      
```

Comment:
we can use material-UI icons, their codes look simpler, we can use their other components as well :)
- installing (icons): npm install @material-ui/icons
                              npm install @material-ui/core
- example use: import MailOutlineIcon from '@material-ui/icons/MailOutline';
                        
![image](https://user-images.githubusercontent.com/43271693/83180171-9e660680-a155-11ea-9dd6-8c88329b17d0.png)

- finding icons: https://material-ui.com/components/material-icons/#material-icons

Reply:
Oh nice this is much better!